index.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>  
+    
+    <section class="firstPart">
+        <header class="header">
+        <nav class="navbar">
+        <div class="container">
+            <div class="logo"><a href="#"><img src="img/My Logo.png"></a></div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
+    </div>
+</nav>
+</header> 
+      
+        <section class="container1">
+                    <div class="textInfo">
+             <p id="intro">I’m Amino </br> Glad to see you!</p>
+                
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div> 
+                </section>   
+                
+    <button class="con">LET'S CONNECT </button>
+    <div class="social"> 
+        <ul class="hero">    
+        <li><a href="#" class="scIcon"><img src="/img/social/Twitter (2).svg" alt=""></a></li>
+    
+        <li><a href="#" class="scIcon"><img src="img/social/Linkedin icon (1).svg" alt=""></a></li>
+    
+        <li><a href="#" class="scIcon"><img src="img/social/Medium (2).svg" alt=""></a></li>
+    
+        <li><a href="#" class="scIcon"><img src="img/social/gihub.svg" alt=""></a></li>
+    
+        <li><a href="#" class="scIcon"><img src="img/social/angel.svg" alt=""></a></li>
+        </ul>
+     </div>    
```

Comment:
The social media icons use images, which are not accessible to screen readers. It's better to use text and style it with CSS to make it accessible for all users.

Reply:
- [ ] 🙏Please check out the above comment about being polite.

- [OPTIONAL] Please consider using emojis👩‍🦰 in your comments, They make the code review more human-friendly😃